### PR TITLE
Fix errors in the README

### PR DESCRIPTION
**Pull request by benedictflorance**

### CODE REVIEW

**Comment by sonasingh46** [comment]
@benedictflorance Please fix the DCO sign.

**Comment by benedictflorance** [comment]
How do I add the missing DCO sign now? Kindly help.

**Comment by benedictflorance** [comment]
DCO sign has been fixed.

File: docs/FAQ.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ If you have a Kubernetes environment, you can deploy OpenEBS using the following
 
 `kubectl apply -f https://openebs.github.io/charts/openebs-operator.yaml`
 
-You can then begin running a workload against OpenEBS. There are large and growing number of workloads that have storage classes that use OpenEBS. You need not use these specific storage classes. However, they may be helpful as they save time and allow for per workload customization. You can join the Slack channel at: https://openebs-community.slack.com.
+You can then begin running a workload against OpenEBS. There is large and growing number of workloads that have storage classes that use OpenEBS. You need not use these specific storage classes. However, they may be helpful as they save time and allow for per workload customization. You can join the Slack channel at: https://openebs-community.slack.com.
```
[the remark this answers is not in the document]
Comment (benedictflorance) [reply docs/FAQ.md]: The plural form of 'are' does not seem to agree with the singular subject 'large and growing number'?

**Pull request merged**